src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -52,7 +52,7 @@ def run_pf_command(*args, cwd=None):
 
 
 @pytest.mark.usefixtures(
-    "use_secrets_config_file", "setup_local_connection", "install_custom_tool_pkg", "recording_injection"
+    "use_secrets_config_file", "recording_injection", "setup_local_connection", "install_custom_tool_pkg"
```

Comment:
why change this order?

Reply:
record injection should come before setup_local_connection, since it will hijack Connection class in this patch.